counsel.el

```diff
@@ -2466,7 +2466,14 @@ It applies no filtering to ivy--all-candidates."
     (insert (format "-*- mode:grep; default-directory: %S -*-\n\n\n" default-directory))
     (insert (format "%d candidates:\n" (length ivy--all-candidates)))
     (ivy--occur-insert-lines
-     (mapcar (lambda (cand) (concat "./" prepend cand)) ivy--all-candidates))))
+     (mapcar (lambda (cand)
+               (save-match-data
+                 (if (string-match
+                      ;; Remove column indicators if present
+                      "\\(?1:[^\n:]+?[^\n/:]\\):[\t ]*\\(?3:[1-9][0-9]*\\)[\t ]*:\\(?3:[1-9][0-9]*:\\)" cand)
```

Comment:
Why are there three explicitly numbered groups, when only the last one is going to be used (when there are multiple capture groups with the same number, only the right-most one is used)?

Reply:
I borrowed that from grep.el. let's rework the code after confirmation that the fix works.